workflow-bot-app/src/test/java/com/symphony/bdk/workflow/utils/UtilityFunctionsMapperTest.java

```diff
@@ -49,4 +51,47 @@ void jsonEmptyTest() {
     assertThat(json.toString()).isEmpty();
   }
 
+  @Test
+  void encodeQueryParametersTest() {
```

Comment:
could you add a test with

https://www.wdk.symphony.com:8080/path1/path2?key1=value1&key1=value2 as the URL?